rust/tool/systemd/src/main.rs

```diff
@@ -1,16 +1,9 @@
 mod cli;
 mod esp;
-mod gc;
-mod generation;
 mod install;
-mod os_release;
-mod pe;
-mod signature;
-mod systemd;
-mod utils;
+mod version;
 
 use clap::Parser;
-
```

Comment:
nit: please keep this line it separates external dependencies from internal dependencies.